### DISCUSSION TITLE
Google Assistant Compatibility

### DISCUSSION THREAD

**Discussion by ricardonlo** [PaulWoitaschek/Voice#1400] [Ideas]
I'm new here, do you guys think it is something worth/able to do ?

**Comment by marksz** [comment]
I would really love to be able to ask any Google speaker in my house to "ask Voice to play my current audiobook on all speakers"!

It might get confused though as I have two apps called Voice, as Google Voice is just listed as Voice in my app list.
I suspect this would not be trivial to add.

It already works on a Bluetooth speaker to say pause and resume after starting Voice playing from the phone.  I'll often do this in my bathroom before a swim, so I can just say "Hey Google, resume" to start my book playing without getting my phone wet when I shower afterwards.
